src/context/ActionSidebarContext/ActionSidebarContextProvider.tsx

```diff
@@ -41,7 +42,12 @@ const OPEN_ACTION_PANEL_EVENT: AnalyticsEvent = {
 const ActionSidebarContextProvider: FC<PropsWithChildren> = ({ children }) => {
   const [actionSidebarInitialValues, setActionSidebarInitialValues] =
     useState<FieldValues>();
+  const [isEditMode, setIsEditMode] = useState(false);
   const cancelModalToggle = useToggle();
+  const cancelEditModalToggle = useToggle();
+  const [editValues, setEditValues] = useState<
+    ExpenditurePayoutFieldValue[] | undefined
+  >();
```

Comment:
Could that be moved to expenditure-specific component, such as PaymentBuilderEdit?